### DISCUSSION TITLE
C++ interop

### DISCUSSION THREAD

**Discussion by Jimmy2099** [vlang/v#18690] [Ideas]
### Describe the feature

according to this https://vlang.io/compare
Webpage 

it seem v don’t have lots of advantages when comparing with carbon lang
Specially carbon can do c++ interop in  source code level and don’t need manual  bindings or trancode like c to v translator 

### Use Case

C++ interop 

### Acknowledgements

- [X] I may be able to implement this feature request
- [X] This feature might incur a breaking change

### Version used

1.0

### Environment details (OS name and version, etc.)

No

**Comment by medvednikov** [comment]
C/C++ interop on source level without wrapper generation is planned for 0.5, I've added it to the roadmap.

**Comment by Jimmy2099** [comment]
> C/C++ interop on source level without wrapper generation is planned for 0.5, I've added it to the roadmap.

really hope this feature can be released earlier than carbon lang, so that it may gain a preemptive advantage in the competition
